src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py

```diff
@@ -195,6 +200,12 @@ def readFWJRForErrors(self, jobList):
         cooloffJobs = []
         passJobs = []
         exhaustJobs = []
+
+        if self.reqAuxDB is None:
+            exitCodesNoRetry = self.config.ErrorHandler.failureExitCodes
```

Comment:
Is T0 going to overwrite the default WMAgent configuration? Otherwise this line will crash since that config section won't have `failureExitCodes` attr. I'll make a new PR with my suggestions instead of writing them here :)